app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ActiveRecord::Base
+  include AsUserAssociationExtension
+
+  belongs_to :photo
+  belongs_to :user
+  validates_presence_of :user
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.